source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOBaseCardElement.mm

```diff
@@ -0,0 +1,34 @@
+//
+//  ACOBaseCardElement
+//  ACOBaseCardElement.mm
+//
+//  Copyright © 2018 Microsoft. All rights reserved.
+//
+#import <Foundation/Foundation.h>
+#import "ACOBaseCardElement.h"
+#import "BaseCardElement.h"
+
+using namespace AdaptiveCards;
+
+@implementation ACOBaseCardElement
+{
+    std::shared_ptr<BaseCardElement> _elem;
+}
+
+- (instancetype)init
+{
+    self = [super init];
+    return self;
+}
+
+- (std::shared_ptr<BaseCardElement>)getElem
```

Comment:
>getElem [](start = 36, length = 7)

Minor, but accessors in Objective-C aren't normally prefixed with `get`. The method name would just be `elem`, or even better, `element`.

Reply:
thank you

---
In reply to: [169506601](https://github.com/Microsoft/AdaptiveCards/pull/1143#discussion_r169506601) [](ancestors = 169506601)